scripts/diagnostics.sh

```diff
@@ -1,145 +1,343 @@
-#!/usr/bin/env bash
-#* This file is part of the MOOSE framework
-#* https://www.mooseframework.org
-#*
-#* All rights reserved, see COPYRIGHT for full restrictions
-#* https://github.com/idaholab/moose/blob/master/COPYRIGHT
-#*
-#* Licensed under LGPL 2.1, please see LICENSE for details
-#* https://www.gnu.org/licenses/lgpl-2.1.html
+#!/bin/bash
+# Script supports Linux (x86_64), Mac (x86_64, Arm)
+export CONDA_VERSION='23.11.0-0'
```

Comment:
can we pull this from another file? or should we just remember to update it

Reply:
I'll make use of `moose/framework/doc/packages_config.yml`